.github/workflows/test.yaml

```diff
@@ -0,0 +1,88 @@
+name: Tests stage - unit test for sniffs
+
+on:
+    pull_request:
+        branches: [ master, develop ]
+    push:
+        branches: [ master ]
+
+jobs:
+    tests:
+        name: Unit tests on PHP ${{ matrix.php-versions }} with PHPCS ${{ matrix.phpcs-versions }} and WPCS ${{ matrix.wpcs-versions }}
```

Comment:
Let's make this less verbose. We know that we're running unit tests since this is the test workflow.

I'd say `PHP X with PHPCS Y/WPCS Z`. This is what displays as the job title in the left hand sidebar, so if  we can make it short, it will be easier to scan to find the exact job we want. Otherwise, the PHPCS and WPCS versions will be ticked away.

Reply:
Good point. The name is cut out atm when checking the actions tab. Will change this 🙂